__mocks__/@gorhom/bottom-sheet.ts

```diff
@@ -0,0 +1,7 @@
+import { ScrollView } from 'react-native'
+
+module.exports = {
+  __esModule: true,
+  ...(jest.requireActual('@gorhom/bottom-sheet') as any),
+  BottomSheetScrollView: ScrollView,
```

Comment:
the BottomSheetScrollView has an internal utility called `getRefNativeTag` that "grab the native tag from the native host component. It only works when the ref is pointing to a native Host component" which is a real pain for unit tests. mocking it to be a RN scrollview seems harmless enough